### PR TITLE
Bug fixes + small additions

### PR DESCRIPTION
**Pull request by alexandrebouchard**
- Fix an edge case in DataSource / Plate underpinnings
- Fix a dependency bug
- Add LogUniform
- Fix COPIES initialization
- Change default: do not run pxviz by default

### CODE REVIEW

File: src/main/java/blang/distributions/LogUniform.bl [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package blang.distributions
+
+model LogUniform {
+  random RealVar realization
+  
+  /** The left end point \(m\) of the interval. \(m \in (\infty, M)\) */
```
Comment (kevinchern) [inline src/main/java/blang/distributions/LogUniform.bl]: Should be (**-**\infty, M)

File: src/main/java/blang/distributions/LogUniform.bl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    logf(min, max) {
+      if (max - min <= 0.0) return NEGATIVE_INFINITY
+      return - log(max - min)
+    }
+    logf(realization, min, max, base) {
+      if (base <= 0) return NEGATIVE_INFINITY
+      val logr = log(realization)
+      val logb = log(base)
+      val transformed = logr / logb
+      if (min <= transformed && transformed <= max) return -logr - log(logb)
```
Comment (kevinchern) [inline src/main/java/blang/distributions/LogUniform.bl]: Might be more robust to have a realization > 0 check before taking log(realization) (?)

Comment (alexandrebouchard) [reply src/main/java/blang/distributions/LogUniform.bl]: Thanks! Just pushed these fixes.

**Comment by alexandrebouchard** [comment]
Thanks!

**Pull request merged**